app/run.py

```diff
@@ -1,5 +1,4 @@
 from __init__ import *
```

Comment:
Lose this line. You should never import form `__init__` import instead from the module that has been initialized with the `__init__.py` file

Reply:
doing that